packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -77,9 +79,23 @@ export const BookEventForm = ({ onCancel }: BookEventFormProps) => {
       return {};
     }
 
+    const querySchema = getBookingResponsesPartialSchema({
+      eventType: {
+        bookingFields: getBookingFieldsWithSystemFields(eventType),
+      },
+      view: rescheduleUid ? "reschedule" : "booking",
+    });
+
+    const parsedQuery = querySchema.parse({
```

Comment:
querySchema knows that guests should be converted to an array and without doing that, FormBuidlerField would crash as it expects an array for guests which is `textList` type